widget/embedded/src/constants/errors.ts

```diff
@@ -29,7 +29,7 @@ export const errorMessages = () => {
       description: i18n.t(
         'The price impact is significantly higher than the allowed amount. If you are sure, continue, otherwise, change the swap.'
       ),
-      confirmMessage: i18n.t('Confirm High price impact'),
+      confirmMessage: i18n.t('Confirm high price impact'),
```

Comment:
Maybe it's not related to this pr but can you check why my changes in liquidity sources doesnt work?

![Recording 2023-11-18 at 15 08 29](https://github.com/rango-exchange/rango-client/assets/99180934/e54c8311-e043-423e-9084-8fdcb212d754)